contracts/BuilderNFTSeasonOneImplementation01.sol

```diff
@@ -237,17 +239,49 @@ contract BuilderNFTSeasonOneImplementation01 is Context, ERC165, IERC1155, IERC1
         return MemoryUtils.getUint256(MemoryUtils.PRICE_INCREMENT_SLOT);
     }
 
-    function uri(uint256 _tokenId) external pure override returns (string memory) {
+    function setUriPrefix(string memory newPrefix) external onlyAdmin {
```

Comment:
Could we please get a single utility method which takes [prefix, suffix] as args?

This will allow us to migrate to a new url format in a single transaction, instead of requiring 2 calls